hnn_core/network.py

```diff
@@ -1185,6 +1193,42 @@ def clear_drives(self):
         self.external_drives = dict()
         self.connectivity = connectivity
 
+    def add_electrode_array(self, name, electrode_pos, sigma=0.3,
+                            method='psa'):
+        """Specify coordinates of an electrode array for LFP recording.
+
+        Parameters
+        ----------
+        name : str
+            Unique name of the array.
+        electrode_pos : tuple | list of tuple
+            Coordinates specifying the position for LFP electrodes in
+            the form of (x, y, z) (in um).
+        sigma : float
+            Extracellular conductivity, in S/m, of the assumed infinite,
+            homogeneous volume conductor that the cell and electrode are in.
+        method : str
+            'psa' (default), i.e., point source approximation or line source
+            approximation, i.e., 'lsa'
+        """
+        from .lfp import LFPArray
```

Comment:
why nested?

Reply:
circular imports are usually a sign of something convoluted about the code logic ... ideally, you should not have to do them (unless you're trying to prevent a hard dependency)